Resources/installer/sql/demo_data.sql

```diff
@@ -608,7 +609,6 @@ INSERT INTO `ezcontentclass_classgroup` VALUES (37,0,3,'Media');
 INSERT INTO `ezcontentclass_classgroup` VALUES (38,0,1,'Content');
 INSERT INTO `ezcontentclass_classgroup` VALUES (39,0,1,'Content');
 INSERT INTO `ezcontentclass_classgroup` VALUES (40,0,1,'Content');
-INSERT INTO `ezcontentclass_classgroup` VALUES (40,1,1,'Content');
```

Comment:
From what I understand is that second `insert` value here:

```sql
INSERT INTO `ezcontentclass_classgroup` VALUES (40,1,1,'Content');
```

is `contentclass_version` and since eZ Publish decided to remove unused version (draft?) it's ok.